json-rpc/docs/type_event.md

```diff
@@ -84,6 +84,26 @@ Event emitted when an account sends a payment.
 | receiver            | string                   | Hex-encoded address of the account whose balance was credited by this deposit. |
 | metadata            | string                   | An optional field that can contain extra metadata for the event (from RawTransaction metadata??). This information can be used by an off-chain API to implement a sub-addressing scheme for a wallet. |
 
+#### compliancekeyrotation
+
+Event emitted when the public key used for dual attestation checking on-chain is rotated.
```

Comment:
Could you update there to mention where to find event key, and looks like only designated_dealer account can have it.